src/Plugin/DataType/LoggerEntry.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\DataType\LoggerEntry.
+ */
+
+namespace Drupal\rules\Plugin\DataType;
+
+use Drupal\Core\TypedData\Plugin\DataType\Map;
+
+/**
+ * The logger entry data type.
```

Comment:
Can we describe what this is? Example "Represents a log entry that is created from system log messages."